src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -3614,6 +3620,9 @@ Give the compiler some way to differentiate the methods. For example, you can gi
   <data name="ERR_BadSpecialByRefLocal" xml:space="preserve">
     <value>Parameters or locals of type '{0}' cannot be declared in async methods or lambda expressions.</value>
   </data>
+  <data name="ERR_BadSpecialByRefIterator" xml:space="preserve">
+    <value>foreach statement cannot operate on variables of type '{0}' in async or iterator methods.</value>
```

Comment:
[https://github.com/dotnet/roslyn/issues/20344](https://github.com/dotnet/roslyn/issues/20344)

---
In reply to: [123129912](https://github.com/dotnet/roslyn/pull/20293#discussion_r123129912) [](ancestors = 123129912,123128084,122807111)